variables.tf

```diff
@@ -47,6 +47,24 @@ variable "alb_listener_priority" {
   type        = number
 }
 
+variable "alb_cogino_pool_arn" {
+  type        = string
+  default     = null
```

Comment:
we could also used empty strings as defaults

Reply:
done